includes/pipeline_page/docs_schema.php

```diff
@@ -73,20 +73,22 @@ function print_param($is_group, $param_id, $param, $is_required=false){
     $param_body = '<div id="'.$param_id.'-body" class="param-docs-body small">'.$description.'</div>';
 
     # Extra group classes
-    $mt = '';
-    $id_cols = 'col-10 col-md-5 col-lg-4 col-xl-3 ';
+    $row_class = 'align-items-center';
+    $id_cols = 'col-10 col-md-5 col-lg-4 col-xl-3 small-h';
+    $h_level = 'h3';
     if($is_group){
-      $mt = 'mt-5';
-      $id_cols = 'col h2';
+      // $row_class = 'mt-3';
+      $row_class = 'align-items-baseline mt-5';
+      $id_cols = 'col pl-0 h2';
+      $h_level = 'h2';
     }
 
     # Build row
     return '
-    <div class="row '.$hidden_class.' params-docs-row border-bottom pt-2 '.$mt.'">
-      <div class="'.$id_cols.'">'.$fa_icon.$h_text.'</div>
-      <div class="col">'.$param_body.'</div>
+    <div class="row '.$hidden_class.' params-docs-row border-bottom d-flex '.$row_class.'">
+      <div class="'.$id_cols.'">'.add_ids_to_headers('<'.$h_level.'>'.$fa_icon.$h_text.'</'.$h_level.'>').'</div>
+      <div class="col">'.$param_body.$help_text.'</div>
```

Comment:
I'm not sure about this change. Help text is likely to be quite long and restricting it to half of the content width seems a bit unnecessary. What was the problem with having it full width?

Reply:
it looked out of context, when expanded the full row width. I like this solution more, because the help-text is an extension of the description section, so it should also be rendered in the same location. One compromise could be to make the help-text the length of the description+labels/help-button columns in a new row (would also prevent the parameter name to move too far away if a long help text is expanded.

Reply:
What about if we use custom styles instead of a card? eg. before:

![image](https://user-images.githubusercontent.com/465550/88634860-3715f580-d0b7-11ea-9a41-20e2f91512c3.png)

---

After:

![image](https://user-images.githubusercontent.com/465550/88635026-72b0bf80-d0b7-11ea-8831-a6b00d8da5fd.png)

---

I think that it's realistic to assume that some pipeline authors are going to write a _lot_ of content in here. There has been discussions on Slack about having tables, images, all sorts in this help text. I just think that longer help texts will look odd if kept narrow and be more difficult to read.

Reply:
well, I am not a fan of long text lines in general, because my eyes jump around too much. It doesn't have to be Bringhurst-short, but for example my intuition would be to cut the above at around "qualtity cutoff". I don't see why height of these objects would create a problem, since they were probably already that long in the old style (which had a similar width), especially if it is easily collapsible. Or what is the UX-problem you see there?

Reply:
_"Bringhurst-short"_? 👀 

I guess it's a personal preference thing - I'm the opposite in that I find the second screenshot easier to read. Although no schema have it yet, I'm also worried that tables and other rich content will be wide and simply not fit in the narrower box. Making it full-width gives us the best chance of accommodating whatever people throw at us.

I also like that the UI doesn't change for the parameter name, as there is no vertical change. This can also be achieved this without being full-width, but we'll have a big chunk of empty whitespace on the left then.

Reply:
Bringhurst-short = 45 to 75 characters https://smad.jmu.edu/shen/webtype/linelength.html

Okay, we can go back to full width help-text. I prefer the light background option to the card one.

Reply:
Nice! Learn something new every day 😅 

We can always meet in the middle if you want - stop the vertical height changing on the param id and make it a little wider on the right so that it flows underneath the Help button. I don't have very strong feelings on the subject.

Light background option = the light grey background?